src/util/fs.js

```diff
@@ -454,3 +455,33 @@ export async function getFileSizeOnDisk(loc: string): Promise<number> {
 export function normalizeOS(body: string): string {
   return body.replace(/\r\n/g, '\n');
 }
+
+const cr = new Buffer('\r', 'utf8')[0];
+const lf = new Buffer('\n', 'utf8')[0];
+
+async function getEolFromFile(path: string) : Promise<string | void>  {
+  let buffer;
+  try {
+    buffer = await readFileBuffer(path);
```

Comment:
I thought about it, but if the file is inaccessible for some other reason - is it OK to just let it fail here? Will this be handled on a higher level?